README.md

```diff
@@ -94,13 +103,25 @@ To begin benchmarking your first embodied agent with NETT, please be aware:
 
 ### Running a NETT
 
-1. **Download or Create the Unity Executable**
+1. **Download the Unity Executable**
+
+   You can download and unzip pre-made Unity executables from [here](https://origins-of-intelligence.com/environments/). An executable is required to run the virtual experiments.
 
-   Download and unzip a pre-made Unity executable from [here](https://origins-of-intelligence.com/environments/). The executable is required to run the virtual environment.
+   If you are running this pipeline on a server and you need to copy the executable folder from your local machine to the server, use the following:
+
+   ```bash
+   scp -r \local\path\to\executable\folder  \server\parth\to\directory\for\executable\folder\
+   ```
+   *(Note: if you are a member of our lab using one of our lab servers, the server path will be: your_username@server_name.luddy.indiana.edu:/home/your_username)*
 
 2. **Import NETT Components**
 
-   Start by importing the NETT framework components - `Brain`, `Body`, and `Environment`, alongside the main `NETT` class.
+   Open python.
+   ```bash
+   python
+   ```
```

Comment:
If we are showing them how to run the code in python, we should probably let them know that they can run it in a file, which would be the preferred/easier method.
```suggestion
   From here, everything will be done in Python. Either open the interpreter in interactive mode by running `python` or write the code into a python file, which can be run with `python <FILE>.py`
```

Reply:
This change works for me!